### ISSUE TITLE
Replace buffer with b4a breaks the support of ram+hypercore for the browser

### ISSUE THREAD

**Issue by tinchoz49** [random-access-storage/random-access-memory#10]
The `uint64be` module used by hypercore expect a Buffer and it receives a UInt8Array from b4a. Maybe a better option to fix this is to change `uint64be` to work with Uint8Array instead of buffers.

```
index.js?5840:24 Uncaught TypeError: buf.readUInt32BE is not a function
    at Object.exports.decode (index.js?5840:24)
    at Request.eval [as _callback] (storage.js?f5ce:173)
    at Request.callback (index.js?9955:174)
    at eval (index.js?9955:269)
```

**Comment by mafintosh** [comment]
Reverted in 3.1.4

**Comment by mafintosh** [comment]
... and published 4.0.0 with the uint8array
